.github/workflows/ci-c.yml

```diff
@@ -0,0 +1,109 @@
+name: Build and test C
+
+on:
+  push:
+    branches: [ master, gvm-libs-20.08, gvm-libs-21.04 ]
+  pull_request:
+    branches: [ master, gvm-libs-20.08, gvm-libs-21.04 ]
+
+
+jobs:
+  c_format_check:
+    name: Check C Source Code Formatting
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - name: Check Source Format
+        run: |
+          clang-format -i -style=file base/*.{c,h} boreas/*.{c,h} \
+          gmp/*.{c,h} osp/*.{c,h} util/*.{c,h} && git diff --exit-code
+  test_units:
+    name: Unit Tests
+    runs-on: 'ubuntu-latest'
+    container: greenbone/build-env-gvm-libs-master-debian-buster-gcc-test
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - name: Configure and test gvm-libs
+        run: |
+          mkdir build && cd build/ && cmake -DCMAKE_BUILD_TYPE=Release \
+          -DBUILD_TESTS=1 .. && make install && make tests && \
+          CTEST_OUTPUT_ON_FAILURE=1 make test
+        working-directory: ${{ github.WORKSPACE }}
+  build_gcc_core:
+    name: Build gvm-libs with gcc
+    runs-on: 'ubuntu-latest'
+    container: greenbone/build-env-gvm-libs-master-debian-buster-gcc-test
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - name: Configure and Compile gvm-libs
+        run: |
+          mkdir build && cd build/ && cmake \
+          -DCMAKE_BUILD_TYPE=Release .. && make install
+        working-directory: ${{ github.WORKSPACE }}
+  scan_build:
+    name: Scan-build gvm-libs with clang
+    runs-on: 'ubuntu-latest'
+    container: greenbone/build-env-gvm-libs-master-debian-buster-clang-core
```

Comment:
Is that the correct docker image for the 21.04 branch?

Reply:
In that docker image nothing with a version is installed. So it is valid for all versions